lib/uplinkc/testdata/object_test.c

```diff
@@ -187,6 +187,37 @@ void handle_project(ProjectRef project) {
             free_downloader(downloader);
         }
 
+        { // download range
+            long start = 100 + (i+1);
+            long limit = 1024 * (i+1);
+            DownloaderRef downloader = download_range(bucket, object_paths[i], start, limit, err);
+            require_noerror(*err);
+
+            uint8_t *downloaded_data = malloc(limit);
+            memset(downloaded_data, '\0', limit);
```

Comment:
any reason not to use `calloc` instead of `malloc` + `memset`?

google turned this up: https://stackoverflow.com/questions/2688466/why-mallocmemset-is-slower-than-calloc

Reply:
hmm another question to the author :) but I guess it doesn't matter too much in test.